### PR TITLE
manolo tlc arc d13 13_1 to 13_1b

### PR DESCRIPTION
**Pull request by nolo16**

### CODE REVIEW

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 1]
```diff
@@ -616,7 +619,7 @@ It's easy to understand how SHIKI would feel about you living in his place while
 [sha:dde1db232f9e8078f37f2e1f82cb82878dcee161]{
 -- Page 129, Offset 32028.
 -- 「……シキにとって、俺は自分の名前を使って遠野シキになりすましているニセモノなわけか」
-"...So to SHIKI, I'm just an impostor who's using my own name to impersonate him."
+"...So to SHIKI, I'm just an impostor who's using his own name to impersonate him."
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:622]: drop "own"

Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:622]: drop "own"

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 2]
```diff
@@ -751,7 +754,7 @@ That person would always be holding back this sort of pain while showing me a br
 [sha:59706bb975c98b6960beec7e7f928b9dc9ae3af0]{
 -- Page 139, Offset 32055.
 -- 「……わかりません。どうしてそこまで彼女に肩入れするんですか。アルクェイドは吸血鬼です。遠野くんとは違うモノじゃないですか」
-"...I don't get it. Why do you want to help her that much? Arcueid is a vampire. Wouldn't that make her different from you?"
+"...I don't get it. Why do you want to help her that much? Arcueid is a vampire. She's a fundamentally different being than you."
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:757]: different being.

Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:757]: different being.

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 3]
```diff
@@ -826,17 +829,17 @@ I'm not sure how helpful I'd be in battle, but it's much more preferable than le
...
 -- Page 147, Offset 32070.
 -- 「単純な足し算ですから。
-"It's simple maths.
+"It's simple math.
 }
 [sha:391ef63065094136303bbcbdb1086b8d8f37c099]{
 -- Page 147, Offset 32071.
 -- 彼女は一度決壊した吸血衝動を抑えて活動している。その為に能力はさらに低下し、ロアの半分にも満たないでしょう。
-Right now, she's trying to hold back her blood-sucking urges that's already been breached once. As a result, her abilities have degraded to the point where it isn't even half of Roa's.
+Right now, she's trying to hold back her blood-sucking urges that've already been unleashed once. As a result, her abilities have degraded to the point where they aren't even half of Roa's.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:837]: deja vu

Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:837]: deja vu

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 4]
```diff
@@ -1021,7 +1024,7 @@ My breathing turns intense just from sitting up on the bed.
 [sha:b691059efbdba221115681a4c9907b53332e8640]{
 -- Page 162, Offset 32109.
 -- こんなんじゃ、とてもじゃないけど学校にたどり着けない。
-If things are like this, then there's no way I can go to the school.
+The way things are now, there's no way I'll be able to go tothe school.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1027]: to get to the school

Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1027]: to get to the school

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 5]
```diff
@@ -1046,7 +1049,7 @@ If it's about Roa, he will be taken care of in a couple of days."
 [sha:18b80606a95832077d7a6e7adfd3025076bcf61b]{
 -- Page 164, Offset 32114.
 -- 「ロアの転生体が特定できましたから、法王庁……えっと、わたしたちの本拠地ですけど、そこに要請が通ります。
-"Since we've identified Roa's reincarnate, the Vatican... well, although it may be our headquarters, we would finally be able to send a request.
+"Since we've identified Roa's reincarnate, we'll finally be able to request aid from the Vatican... uhhh, our headquarters.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1052]: from the Curia

Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1052]: from the Curia

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 6]
```diff
@@ -1086,7 +1089,8 @@ I put strength into my legs.
 [sha:6a605f261f6bb38b34233bad514baa68e77e7a39]{
 -- Page 167, Offset 32122.
 -- 乱れていく呼吸をなんとか誤魔化しながら、ベッドから立ちあがった。
-I somehow move the disordered breathing to the back of my head and stand up from the bed.
+// fuck you 誤魔化しながら
+I somehow move the disordered breathing to the back of my mind and stand up from the bed.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1093]: Hiding my true state, I somehow move my rough breathing to the back of my mind and

Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1093]: Hiding my true state, I somehow move my rough breathing to the back of my mind and

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 7]
```diff
@@ -1111,7 +1115,7 @@ Didn't they teach you, 'Thou shalt not steal'?"
 [sha:f08af44e79db29064f0ecf4fa19169d45a8ec5d2]{
 -- Page 169, Offset 32127.
 -- そうですね、と答えて、先輩は修道服からナイフを取り出した。
-Answering back with 'You're right', Senpai takes out the knife from her garment.
+Replying 'You're right', Senpai retrieves the knife from her habit.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1118]: Senpai retrieves the knife from her habit in affirmation.

Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1118]: Senpai retrieves the knife from her habit in affirmation.

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 8]
```diff
@@ -1231,17 +1235,17 @@ All this time, she was alone like an idiot.
...
 -- Page 178, Offset 32152.
 -- だから、ただ、知らせたかった。
-That's why I just wanted to let her know.
+That's why I just wanted to show her.
 }
 [sha:fe06b56c819f360e54ac0aae609ec49acac791cc]{
 -- Page 178, Offset 32153.
 -- 世の中には沢山の出来事があって、その大半は無意味で無駄なコトだけど。
-That there are plenty of things to do in this world, most of which have no point or deeper meaning.
+That there are plenty of things to do in this world, most of which are pointless and meaningless.
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1248]: most of which are without meaning

Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1248]: most of which are without meaning

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 9]
```diff
@@ -1266,7 +1270,8 @@ Even though she smiles so happily, I wanted to tell her that anyone could obtain
 [sha:4d7452f688006f81ed0f62bbc404f4dc8de27ea6]{
 -- Page 180, Offset 32158.
 -- もっと、つまらない悩みなんて消し飛ぶぐらい楽しい事があるんだって、何回でも連れ回したかった。
-That there are even more fun things in this world that can make her pointless troubles disappear, and I wanted to take her around over and over again.
+// i dont really know what to do with 何回でも連れ回したかった
+There are even more fun things in this world that can make her pointless troubles disappear, .
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1274]: the previous version isn't that bad for 何回でも連れ回したかった. I don't have any improvements

Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1274]: the previous version isn't that bad for 何回でも連れ回したかった. I don't have any improvements

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 10]
```diff
@@ -1306,6 +1311,7 @@ I have to make her even happier for all the times she hadn't been saved. It woul
 [sha:621584ec425a2d4c020488b67ffc8d5914c6431d]{
 -- Page 182, Offset 32166.
 -- だいたい、あいつが独りでなくなる方法なんて簡単すぎる。そんな事で悩んでるとか、馬鹿みたいだ」
+// im sure 簡単 can be improved but i am totally blanking
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1314]: meh, its fine

Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1314]: meh, its fine

File: script/Arcueid/Day 13/13_00_ARC13_1B.txt [hunk 11]
```diff
@@ -1636,6 +1642,7 @@ No, of course not.
 [sha:db47cc9d6e32e9e452ce99a8a05907fb771c16fd]{
 -- Page 208, Offset 32233.
 -- 「知るか。言う事をきかないなら、ここでメチャクチャに暴れてやる」
+// CRINGE MOMENT
```
Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1645]: this sounds god awful in english. Maybe the equally cringe but less awkward "I'll fuck you up."

Comment (Avuxo) [inline script/Arcueid/Day 13/13_00_ARC13_1B.txt:1645]: :aokoheh:

**Review by Avuxo — APPROVED** [review]
I can tell from your comments that these files weren't fun. Otsukaresama deshita

File: script/Arcueid/Day 13/13_00_ARC13_1.txt [hunk 12]
```diff
@@ -696,12 +700,13 @@ But there must have been something else that time. I feel like there's something
 [sha:133a1d539ed1b78d8675aaaa9c57cf10954ee440]{
 -- Page 48, Offset 31807.
 -- もう取り戻せない記憶の中に、俺はこの手で<何|・><か|・><の|・><命|・><を|・><断|・><ち|・><き|・><っ|・><た|・>確信がある。
-Within those memories that I can no longer remember, I'm certain that I took the life of something with these hands.
+// thoughts on italics for something?
+Within those memories that I can no longer regain, I'm certain that with these hands, I took the life of something.
```
[the remark this answers is not in the document]
Comment (nolo16) [reply script/Arcueid/Day 13/13_00_ARC13_1.txt:704]: i meant for "something" my b

[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Arcueid/Day 13/13_00_ARC13_1.txt:704]: Yeah, thats fine in italics I suppose

File: script/Arcueid/Day 13/13_00_ARC13_1.txt [hunk 13]
```diff
@@ -171,6 +172,8 @@ Akiha turns her eyes away from the awkwardness of some sort.
 [sha:6b1e5a3e79cc65cf3f99a87e3bfc2843ee2e36b6]{
 -- Page 11, Offset 31702.
 -- あいつに―――<四|ロ><季|ア>にやられたんだ、とは言えない。
+// please kill me now
+// is this current one fine or should shiki be incorporated
```
[the remark this answers is not in the document]
Comment (nolo16) [reply script/Arcueid/Day 13/13_00_ARC13_1.txt:176]: if no one has any suggestions for this line ill just put it in ruby....?

[the remark this answers is not in the document]
Comment (Avuxo) [reply script/Arcueid/Day 13/13_00_ARC13_1.txt:176]: ruby is fine here

**Pull request merged**